azure-pipelines-yarn-template.yml

```diff
@@ -0,0 +1,26 @@
+jobs:
+- job: ${{ parameters.name }}
+  pool:
+    vmImage: ${{ parameters.vmImage }}
+  strategy:
+    matrix:
+      node_11_x:
+        node_version: 11.x
+      node_10_x:
+        node_version: 10.x
+      node_9_x:
+        node_version: 9.x
+      node_8_x:
+        node_version: 8.x
+      node_6_x:
+        node_version: 6.x
+    maxParallel: 5
+  steps:
+  - task: NodeTool@0
+    inputs:
+      versionSpec: $(node_version)
+    displayName: Install Node.js
+  - bash: yarn install --ignore-engines
+    displayName: Install dependencies
+  - bash: yarn lint:standard && yarn lint:typescript
+    displayName: Run linter
```

Comment:
maybe we should be running the tests as well?

Reply:
Yes, sure.